src/convertToObject.js

```diff
@@ -1,18 +1,57 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
+const stylesString = `
+  background-color:      #fff;
+-webkit-border-radius: 5px;
+  border-radius     : 5px;
+  border: 1px solid #e8e8e8;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  clear   : both       ;
+  cursor: pointer;
+  float: left;
+  font-family: inherit;
+      font-size: 14px;
+  font-weight: 400;
+  height: 42px;
+  line-height:    40px;
+  outline: 0;
+  padding-left    : 18px;
+  padding-right: 30px;
+  ;
+
+  ;
+  position: relative;
+
+
+  text-align: left !important;
+  -webkit-transition: all .2s ease-in-out;
+  transition: all .2s ease-in-out;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+
+  white-space: nowrap;
+  width: auto;
+`;
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';\n')
+    .reduce((accum, item) => {
+      const [key, value] = item.split(':');
+
+      if (key === undefined || value === undefined) {
+        return accum;
+      }
+
+      accum[key.trim()] = value.trim();
+
+      return accum;
+    }, {});
 }
 
+convertToObject(stylesString);
```

Comment:
The function is being called unnecessarily. Remove this line, the function should be exported and used elsewhere, not in this file.